### PR TITLE
Update moderator/conduct team contacts

### PR DESCRIPTION
**Pull request by jonmeow**

### CODE REVIEW

**Review by Pixep — APPROVED** [review]
LGTM assuming the email address is correct, which I cannot verify :). Might need an OK from the mod team.

**Comment by jonmeow** [comment]
> LGTM assuming the email address is correct, which I cannot verify :). Might need an OK from the mod team.

Definitely looking for approval from at least one lead here. :)

**Review by chandlerc — APPROVED** [review]
LGTM, this seems like a a good, simple improvement and gets everything up-to-date!

**Pull request merged**
